benchmark/util/get-callsite.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const common = require('../common');
+const { getCallSite } = require('node:util');
+const assert = require('node:assert');
+
+const bench = common.createBenchmark(main, {
+  n: [1e6],
+  method: ['ErrorCallSite', 'ErrorCallSiteSerialized', 'CPP'],
+});
+
+function ErrorGetCallSite() {
```

Comment:
Although we are using `Error` to get a stack-trace, isn't the purpose of this utility to be used for anything besides errors? Wouldn't make sense to then name these functions to something more appropriate?

Reply:
This is specifically to benchmark approaches, Error is one of them.